packages/cli/src/commands/init/init.js

```diff
@@ -90,6 +93,90 @@ async function createFromExternalTemplate(
   }
 }
 
+async function installPods(loader: typeof Ora) {
+  process.chdir('ios');
+
+  const hasPods = await fs.pathExists('Podfile');
+
+  if (!hasPods) {
+    return;
+  }
+
+  try {
+    await commandExists('pod');
+  } catch (err) {
+    loader.succeed();
+
+    const {shouldInstallCocoaPods} = await inquirer.prompt([
+      {
+        type: 'confirm',
+        name: 'shouldInstallCocoaPods',
+        message: 'CocoaPods is not installed, Do you want to install it?',
+      },
+    ]);
+
+    if (shouldInstallCocoaPods) {
+      try {
+        await execa('sudo', ['gem', 'install', 'cocoapods'], {
+          stdio: 'pipe',
+        });
+      } catch (err) {
+        throw new CLIError(
+          'Error occurred while trying to install CocoaPods, please install it manually.',
+          err,
+        );
+      }
+
+      try {
+        loader.start('Installing pods');
+      } catch (err) {
+        throw new CLIError(
+          'Failed to run "pod install", please try to run it manually.',
```

Comment:
We don't run `pod install` here (check the `try` clause). I think you had a issue while refactoring, because correct code is on line 141 :)

Reply:
Done!